service/src/main/java/uk/gov/hmcts/reform/fpl/service/OrganisationService.java

```diff
@@ -32,16 +32,14 @@
     private final RequestData requestData;
 
     public Set<String> findUserIdsInSameOrganisation(String localAuthorityCode) {
-
         try {
-            return Set.copyOf(getUsersFromSameOrganisationBasedOnAppConfig(localAuthorityCode));
-        } catch (UnknownLocalAuthorityCodeException ex) {
+            return Set.copyOf(getUsersFromSameOrganisationBasedOnReferenceData(requestData.authorisation()));
+        } catch (FeignException e) {
```

Comment:
exception is not even logged, we should be able to distinguish between situation where theres is failure because org is not configured and between failures like Service unavailable.

Anyway I would add more logs which helps us tell, which path was used